src/FSharp.Core/async.fs

```diff
@@ -1501,6 +1501,10 @@ type Async =
 
             computation.Invoke newCtxt)
 
+    static member RunImmediate(computation: Async<'T>, ?cancellationToken: CancellationToken) =
```

Comment:
Probably should be preview right?

https://github.com/dotnet/fsharp/blob/bc88b51bec7cd3a76d0e4b2eb2f4b5aa1e8b50ca/src/FSharp.Core/prim-types.fsi#L2571